references/openapi-mock/pipeline.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPT_PATH="$( (cd "$(dirname "$0")" && pwd))"
+
+(cd "$SCRIPT_PATH" && npm i --no-fund)
+(cd "$SCRIPT_PATH" && npm run test)
```

Comment:
The pipeline is supposed to actually run/use the tool 😃 

Please could you update to effectively run what you mention in the README.
You can find the Apigee X env variables to reference in https://github.com/apigee/devrel/blob/main/PIPELINES.md

Reply:
Added integration test (using Jest + Supertest), and invoking from the pipeline.sh